packages/n4s/src/rules/__tests__/isValueOf.test.ts

```diff
@@ -0,0 +1,36 @@
+import { enforce } from 'enforce';
+import { isValueOf, isNotValueOf } from 'isValueOf';
+
+const objectToTest = {
+  a: 'Alpha',
+  b: 'Bravo',
+  c: 'Charlie',
+};
+
+describe('isValueOf tests', () => {
+  describe('When "Bravo" is the value of key "b"', () => {
```

Comment:
The test doesn't care about the specific value, only that it matches, so the test description should not refer to specific value. This is the same for the rest of the tests in this suite.


```suggestion
  describe('When the value exists in the object', () => {
```